tests/system/action/motion/test_create_forwarded.py

```diff
@@ -0,0 +1,174 @@
+from typing import Any, Dict
+
+from openslides_backend.permissions.permissions import Permissions
+from tests.system.action.base import BaseActionTestCase
+
+
+class MotionCreateForwarded(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.permission_test_model: Dict[str, Dict[str, Any]] = {
+            "meeting/1": {"name": "name_XDAddEAW", "committee_id": 53},
+            "meeting/2": {
+                "name": "name_SNLGsvIV",
+                "motions_default_workflow_id": 12,
+                "committee_id": 52,
+            },
+            "motion_workflow/12": {
+                "name": "name_workflow1",
+                "first_state_id": 34,
+                "state_ids": [34],
+            },
+            "motion_state/34": {"name": "name_state34", "meeting_id": 2},
+            "motion/12": {
+                "title": "title_FcnPUXJB",
+                "meeting_id": 1,
+                "state_id": 34,
+            },
+            "committee/52": {"name": "name_EeKbwxpa"},
+            "committee/53": {
+                "name": "name_auSwgfJC",
+                "forward_to_committee_ids": [52],
+            },
+        }
+
+    def test_correct_origin_id_set(self) -> None:
+        self.set_models(
```

Comment:
these looks very copy&pasted. Please clean them up for every test, so only relevant things are set.

Reply:
Add a test_model and use it in most contexts. This reduce the set_model code.